pkg/api/adjustment.go

```diff
@@ -253,6 +253,12 @@ func (a *ContainerAdjustment) SetLinuxCgroupsPath(value string) {
 	a.Linux.CgroupsPath = value
 }
 
+// SetLinuxOomScoreAdj records setting the kernel's Out-Of-Memory (OOM) killer score for a container.
+func (a *ContainerAdjustment) SetLinuxOomScoreAdj(value int) {
```

Comment:
Maybe we could change that like this:
```suggestion
	func (a *ContainerAdjustment) SetLinuxOomScoreAdj(value *int) {
```
In this way, we can unset/ nil the oomscoreadj with the change of generate.go. @mikebrow

Reply:
right .. let's go that route I think.